CHANGELOG.md

```diff
@@ -0,0 +1,92 @@
+<a name="4.0.0"></a>
+## [4.0.0](https://github.com/osquery/osquery/compare/3.3.2...4.0.0) (2019-06-28)
```

Comment:
re: date, I initially used my hodgepodge of a script to grab all `git log` and grep for `fix|ed`, `add|ed`, `table`, `implemented` etc. I will remove the date.

(My hop is in the future and we can automate this somewhat -- but it will always be prone to miss stuff)